src/loader.js

```diff
@@ -696,116 +685,3 @@
     for (var p in descriptors)
       this[p] = descriptors[p];
   }
-
-  function InternalMapPolyfill() {}
```

Comment:
I agree with removing this now, but I think in future we may well recover it in some way as a high-performance polyfill approach for a backwards-compatible loader variant.